### PR TITLE
Homework27: Ekaterina Zavadskaya

### PR DESCRIPTION
**Pull request by KateZavadskaya**
Add hm27

### CODE REVIEW

File: api/tests/test_create.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+            "referralCode": {"type": "string"},
+            "status": {"type": "string"}
+        },
+        "required": ["id", "name", "email", "age",
+                     "phoneNumber", "address", "role",
+                     "referralCode", "status"]
+    }
+    try:
+        validate(instance=response_json, schema=schema)
+        print("JSON-ответ соответствует :) схеме")
```
Comment (Stml89) [inline api/tests/test_create.py]: use `logger` instead of `print`

File: api/tests/test_create.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    response = requests.post(f"{baseUrl}{postUserUrl}",
+                             json=payload,
+                             headers=headers,
+                             timeout=10)
+
+    response_json = response.json()
+    print(response.status_code)
+    print(response_json)
+    assert response.status_code == 200, \
+        f"Ожидался статус-код 200, но получен {response.status_code}"
```
Comment (Stml89) [inline api/tests/test_create.py:46]: move the common part to a separate module, so you can reuse it here:
```

def post(...):
    response = requests.post(...)
    assert response.status_code == expected_code, ....
    return response.json()

def get(...):
    response = requests.get(...)
    ....

def delete...
    ...

....
```

**Review by Stml89 — CHANGES_REQUESTED** [review]
Fix linters

File: api/tests/test_create.py (hunk 2 above)
Comment (KateZavadskaya) [reply api/tests/test_create.py:46]: Sorry, it is more that I can for now.

**Review by Stml89 — APPROVED** [review]

**Pull request merged**